src/main/scala/it/pagopa/interop/tenantprocess/api/impl/TenantApiServiceImpl.scala

```diff
@@ -63,6 +67,29 @@ final case class TenantApiServiceImpl(
   )(implicit contexts: Seq[(String, String)], toEntityMarshallerProblem: ToEntityMarshaller[Problem]): Route =
     authorizeInterop(hasPermissions(roles: _*), problemOf(StatusCodes.Forbidden, OperationForbidden))(route)
 
+  override def getProducers(name: Option[String], offset: Int, limit: Int)(implicit
+    contexts: Seq[(String, String)],
+    toEntityMarshallerTenants: ToEntityMarshaller[Tenants]
+  ): Route = {
+    implicit val problemMarshaller: ToEntityMarshaller[Problem] = entityMarshallerProblem
```

Comment:
Why wasn't this declared in the signature? Is it not necessary?

Reply:
yes, but on a second thought the endpoint could return a 403.
I'll add it to the spec to avoid this import, thanks for pointing it!
(I'll make the change in [this PR](https://github.com/pagopa/interop-be-tenant-process/pull/37) to limit conflicts)

Reply:
🙇

Reply:
done https://github.com/pagopa/interop-be-tenant-process/pull/37/commits/cdd3ba9c00775a11a59f1dc62671780cf26ddc7b